commands/post-install.ts

```diff
@@ -0,0 +1,31 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+
+import options = require("../../common/options");
+
+export class PostInstallCommand implements ICommand {
+	constructor(private $autoCompletionService: IAutoCompletionService,
+		private $fs: IFileSystem,
+		private $staticConfig: IStaticConfig) {
+	}
+
+	public disableAnalytics = true;
+	public allowedParameters: ICommandParameter[] = [];
+
+	public execute(args: string[]): IFuture<void> {
+		return (() => {
+			if(process.platform !== "win32") {
+				// when running under 'sudo' we create ~/.appbuilder-cli with wrong owner (root) and it is no longer accessible for the user initiating the installation
```

Comment:
comment is talking about .appbuilder-cli, but the code is used for nativescript-cli as well